db/patch71.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO `joindin`.`talk_links` (`talk_id`, `talk_type`, `url`)
+SELECT
+  id,
+  1,
+  slides_link
+FROM
+  talks
+WHERE slides_link <> ''
```

Comment:
This is currently throwing a syntax error in mySQL - think it's just a missing semi-colon however.